src/platform/qt/DebuggerConsoleController.cpp

```diff
@@ -101,6 +100,8 @@ void DebuggerConsoleController::lineAppend(struct CLIDebuggerBackend* be, const
 }
 
 const char* DebuggerConsoleController::historyLast(struct CLIDebuggerBackend* be, size_t* len) {
+	Q_UNUSED(len)
```

Comment:
I get the impression that this parameter is supposed to be used, but I'd need to check how. Squelching it is probably not the solution here.